.eslintrc

```diff
@@ -1,6 +1,51 @@
 {
-  "extends": "hypothesis",
+  "extends": ["hypothesis", "plugin:@typescript-eslint/recommended"],
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint"],
+  "rules": {
```

Comment:
These @typescript-eslint rules are currently duplicated across several repositories and should be folded into the eslint-config-hypothesis package in future.

Reply:
Yes, please!